src/reporter/delete.py

```diff
@@ -46,6 +65,7 @@ def delete_entities(entity_type, from_date=None, to_date=None,
                                         fiware_service=fiware_s(),
                                         fiware_servicepath=fiware_sp(),)
 
```

Comment:
um, isn't this `with` block fishy? it looks like the code as it was before your change? could it be a result of a rebase gone wrong? if you could please double-check what the heck is going on here, but I think we should zap this entire block, i.e. delete lines 55 through 67?

Reply:
Hi @c0c0n3 , I have updated changes to PR . Thanks

Reply:
@daminichopra that's great, thx soooo much!